spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -50,6 +50,8 @@
 
 	private String defaultBinder;
 
+	private String[] outboundChannelNames = new String[0];
```

Comment:
This property might be slightly confusing, since it does not include the outputs of `Source`, `Processor` or anything else specified via `@EnableBinding`, which definitely *are* output channels (granted, this feature is intended to be used mainly by Sinks). This is very specific to dynamic binding so I think we should find a way to express that.